src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultItems.props

```diff
@@ -33,14 +41,4 @@
     />
   </ItemGroup>
```

Comment:
The `@(FrameworkReference)` in this file breaks builds for `netstandard2.0` libraries:

    Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(324,5): error NETSDK1073: The FrameworkReference 'Microsoft.Android' was not recognized

I'll help look into this.

Reply:
The condition on the ItemGroup also needs to check that the TargetFrameworkIdentifier is Android.

Reply:
I moved things around, so the `@(FrameworkReference)` isn't in `AutoImport.props` at all anymore. I think this will be fine?

Reply:
That should also work.